src/app/components/registration-dialog/registration-dialog.component.ts

```diff
@@ -7,19 +9,24 @@ import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
 })
 export class RegistrationDialogComponent implements OnInit {
   public user: any = {};
+  public passwordConfirm = {};
   @Input() options: any = {
     isOpen: false,
     title: '',
     message: '',
     confirm: false,
-    reject: false
+    reject: false,
+    changed: false,
```

Comment:
Do not leave "Trailing commas" in object. That's TypeScript issue will cause to Fail of production build.